serverlessworkflow/modules/ROOT/pages/index.adoc

```diff
@@ -1,5 +1,13 @@
 = {page-component-title}
 
+{product_name} is a tool for building cloud-native workflow applications. You can use the {product_name} to do the services and events orchestration and choreography. Currently, with {product_name} you can integrate with services and events in your architecture using:
```

Comment:
I agree with @radtriste, here we should probably use `{product_name} {context}` as we are referring to a specific part of Kogito.